.github/workflows/linked-issue-check.yml

```
@@ -0,0 +1,129 @@
+        with:
+          path: generator-jhipster
+          fetch-depth: 2
+      - name: 'SETUP: environment'
+        id: setup
+        uses: ./generator-jhipster/.github/actions/setup
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ steps.setup.outputs.node-version }}
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ steps.setup.outputs.java-version }}
+      - uses: jhipster/actions/restore-cache@v0
+        with:
+          npm: true
+          maven: true
+          gradle: true
+      - name: 'TOOLS: configure tools installed by the system'
+        run: $JHI_SCRIPTS/03-system.sh
+      - name: 'TOOLS: configure git'
+        run: $JHI_SCRIPTS/04-git-config.sh
+      - name: 'Install required NPM version'
+        run: npm install -g npm@${{ steps.setup.outputs.npm-version }} || true
+      #----------------------------------------------------------------------
+      # Install JHipster and generate project+entities
+      #----------------------------------------------------------------------
+      - name: 'GENERATION: install JHipster'
+        run: $JHI_SCRIPTS/10-install-jhipster.sh
+      - name: 'GENERATION: project'
+        id: project
+        run: jhipster from-issue "$(echo ${{ matrix.linked-issue }} | cut -d "#" -f 2)" --no-code-workspace
+      - name: 'GENERATION: jhipster info'
+        run: $JHI_SCRIPTS/14-jhipster-info.sh
+      #----------------------------------------------------------------------
+      # Launch tests
+      #----------------------------------------------------------------------
+      - name: 'PREPARE: npm install'
+        run: npm install
+        timeout-minutes: 7
+      - name: 'TESTS: backend'
+        id: backend
+        run: npm run ci:backend:test --if-present
+        timeout-minutes: 15
+      - name: 'TESTS: frontend'
+        id: frontend
+        run: npm run ci:frontend:test --if-present
+        timeout-minutes: 15
+      - name: 'TESTS: packaging'
+        run: npm run ci:e2e:package --if-present
+        timeout-minutes: 12
+      - name: 'TESTS: Start docker compose containers for e2e tests'
+        run: npm run ci:e2e:prepare --if-present
+        timeout-minutes: 5
+      - name: 'E2E: Run'
+        id: e2e
+        run: npm run ci:e2e:run --if-present
+        timeout-minutes: 15
+      - name: 'BACKEND: Store failure logs'
+        uses: actions/upload-artifact@v3
+        if: always() && steps.backend.outcome == 'failure'
+        with:
+          name: log-${{ matrix.number }}
+          path: app/**/test-results/**/*.xml
+      - name: 'E2E: Store failure screenshots'
+        uses: actions/upload-artifact@v3
+        if: always() && steps.e2e.outcome == 'failure'
+        with:
+          name: screenshots-${{ matrix.number }}
+          path: app/**/cypress/screenshots
+      - name: Dump docker logs
+        if: always()
+        uses: jwalton/gh-docker-logs@v2
```

Add a summary job to be used as required.
```suggestion
        uses: jwalton/gh-docker-logs@v2
  check-closing-issues:
    permissions:
      contents: none
    runs-on: ubuntu-latest
    needs: [applications]
    if: always()
    steps:
      - run: |
          echo '${{ toJSON(needs) }}'
          if [ 'skipped' == '${{ needs.closing-issues.result }}' ] || [ 'success' == '${{ needs.closing-issues.result }}' ] || [ 'closed' == '${{ github.event.action }}' ]; then
            exit 0
          fi
          exit 1
```